### ISSUE TITLE
Let users download generated JS container

### ISSUE THREAD

**Issue by tsteur** [matomo-org/tag-manager#7]
This will be especially useful in case their analytics domain/path contains eg Matomo and is blocked by ad blockers or similar. By downloading the JS file they can embed the generated file manually and reduce risk of the generated container being blocked. Also they could directly minify it and merge it with other JS files to reduce the number of loaded files.

Obviously, when using this method, the JS container file needs to be updated manually on the website after publishing it.

**Comment by peterbo** [comment]
Great idea - also a notice like "tag manager library is out out sync / needs to be updated manually" would be great if it was implemented from external (e.g. a CDN)

**Comment by tsteur** [comment]
Good point. Could even think of features like "sync with a CDN" and could integrate various CDN's (eg we could create one interface for CloudFront and allow plugin developers or contributors to add more)

**Comment by tsteur** [comment]
BTW the easiest way to achieve this is using your own CDN and configure it to point to the Matomo installation the `js` directory for pattern `container*.js` . This way you pretty much don't need to do anything except for setting up CDN once (or you could set up your own proxy like haproxy or in apache etc)
